mobile/lib/models/insight.dart

```diff
@@ -22,43 +22,70 @@ class Insight with EquatableMixin {
     String airQualityMessage = '';
     String forecastMessage = '';
     AirQuality airQuality = airQualityReading.airQuality();
+    String verb = airQualityReading.dateTime.isToday()
+        ? "is"
+        : airQualityReading.dateTime.isYesterday() ||
+                airQualityReading.dateTime.isAPastDate()
+            ? "was"
+            : "might be";
+    String dateAdverb = airQualityReading.dateTime.isYesterday()
+        ? "yesterday"
+        : airQualityReading.dateTime.isTomorrow()
+            ? "tomorrow"
+            : "";
 
-    if (airQualityReading.dateTime.isYesterday()) {
-      airQualityMessage =
-          'The average air quality yesterday in ${airQualityReading.name} was ${airQuality.title}.';
-    } else if (airQualityReading.dateTime.isAPastDate()) {
-      airQualityMessage =
-          'The average air quality in ${airQualityReading.name} was ${airQuality.title}.';
-    } else if (airQualityReading.dateTime.isToday()) {
-      airQualityMessage =
-          'The average air quality in ${airQualityReading.name} is currently ${airQuality.title}.';
+    switch (airQuality.title) {
```

Comment:
Thanks @BenjaminSsempala . You can Switch airQaulity. For exmaple what you did here https://github.com/airqo-platform/AirQo-frontend/pull/1107#discussion_r1140451489

Reply:
Thanks @NoahNsimbe , Done